build/ci_utils/src/cache/goodie/graalpy.rs

```diff
@@ -0,0 +1,128 @@
+use crate::prelude::*;
+use regex::Regex;
+
+use crate::cache::goodie;
+use crate::cache::goodie::Goodie;
+use crate::cache::Cache;
+use crate::env::known::PATH;
+use crate::github::RepoRef;
+use crate::programs::graalpy::GraalPy as GraalPyProgram;
+
+
+pub const CE_BUILDS_REPOSITORY: RepoRef = RepoRef { owner: "oracle", name: "graalpython" };
+
+#[derive(Clone, Debug)]
+pub struct GraalPy {
+    pub client:  Octocrab,
+    pub version: Version,
+    pub os:      OS,
+    pub arch:    Arch,
+}
+
+fn graalpy_version_from_str(version_string: &str) -> Result<Version> {
+    let line = version_string.lines().find(|line| line.contains("GraalVM CE")).context(
+        "There is a Java environment available but it is not recognizable as GraalVM one.",
```

Comment:
```suggestion
        "There is a Java environment available but it is not recognizable as a GraalVM CE.",
```

We are checking for Graal CE specifically.

I assume if someone runs GraalVM EE, the current message `There is a Java environment available but it is not recognizable as GraalVM one.` would be false (GraalVM EE is still GraalVM).